example/src/demos/MultiView.tsx

```diff
@@ -0,0 +1,168 @@
+import * as THREE from 'three'
+import React, { useRef, useEffect, Suspense, useState } from 'react'
+import { Canvas, useFrame, useThree, createPortal, ReactThreeFiber, Camera, EventManager } from '@react-three/fiber'
+import { useGLTF, useFBO, OrbitControls } from '@react-three/drei'
+import { useCallback } from 'react'
+
+function Window({ index = 1, children, clearColor = 'white', placement = 'topright', ...props }: any) {
+  const ref = useRef<THREE.Mesh>(null!)
+  const { scene: defaultScene, camera: defaultCamera, events, viewport, size } = useThree()
+  const [camera] = useState(() => new THREE.PerspectiveCamera(50, 1, 0.1, 1000))
+  const [scene] = useState(() => new THREE.Scene())
+  const fbo = useFBO()
+
+  clearColor = new THREE.Color(clearColor).convertLinearToSRGB()
```

Comment:
useEffect?

Reply:
It's a bit of a weird pattern, this line

Reply:
I fixed these on the v8 branch.

Reply:
thanks @CodyJasonBennett! as said above, using these demos for testing atm. but when this becomes a drei component it must be cleaned, they wont have render fn stuff in them. but wanted this to be in a beta sooner than later so we may get some real world testing before v8 is released, it's a pretty deep change overall.